app/models/job.ts

```diff
@@ -407,6 +408,24 @@ export class Job extends Record {
     return terminalStates.includes(this.status);
   }
 
+  /**
+   * Check if the job has any STAC data links
+   *
+   * @param transaction - transaction to use for the query
+   * @returns true or false
+   */
+  async hasStacLinks(
+    transaction,
+  ): Promise<boolean> {
+    const links = await transaction('job_links').select()
+      .where({ jobID: this.jobID, rel: 'data' })
+      .whereNotNull('bbox')
```

Comment:
Just want to confirm that for STAC items we only want data links that have bbox, and both a temporal start and end. Is there the concept of an item only having one of temporal start or  end?

Reply:
I think this implies that both should be required if using an interval?

```If there's clearly no meaningful single 'nominal' time, it is allowed to use null instead. In this case it is required to specify a temporal interval with the fields start_datetime and end_datetime from common metadata. For example, if your data is a time-series that covers 100 years, it's not very meaningful to set the datetime to a single timestamp as it would not be found in most searches that searches for a decade of data in that period although the Item actually covers the decade. ```

Reply:
https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md